### DISCUSSION TITLE
Accidental Deletion of my Website on Azure Static Web App.

### DISCUSSION THREAD

**Discussion by sanskritisahu21** [microsoft/SATechnicalOnboarding#99] [Questions]
I recently completed the technical onboarding process for beta Microsoft learn student ambassador and filled the technical onboarding form successfully. After filling the form, unfortunately I made an error by unintentionally deleting my website from the Azure Static Web App. As a result, my website is not accessible, and regrettably, I did not create a backup.

I am reaching out to seek assistance and guidance on the steps I should take to become beta MLSA Or to recover my website.

**Comment by AnthonyByansi** [comment]
hi @sanskritisahu21 Well, it is possible to re-provision your website if you still have the repository in your github account. That would be the same process. You need to delete the workflow directory in your repository and repeat the process
